assets/css/front.css

```diff
@@ -22,11 +22,17 @@
  * SOFTWARE.
  */
 
-#payment .payment_methods .onpay_card_logos {
+ #payment .payment_methods .onpay_card_logos {
     display: block;
 }
 
-#payment .payment_methods .onpay_card_logos > img {
+#payment .payment_methods li.wc_payment_method.payment_method_onpay_card img,
+#payment .payment_methods li.wc_payment_method.payment_method_onpay_mobilepay img,
+#payment .payment_methods li.wc_payment_method.payment_method_onpay_viabill img {
```

Comment:
I think you should strive to make this selector a lot less specific, to allow for themes to override the style

Reply:
This selector has now been made so vague that it only should only apply when using a theme that does dot not have any styling for these woocommerce payment method logos.